pkgs/games/freeciv/default.nix

```diff
@@ -33,7 +33,7 @@ in stdenv.mkDerivation rec {
 
   buildInputs = [ lua5_3 zlib bzip2 curl xz gettext libiconv ]
     ++ optionals sdlClient [ SDL SDL_mixer SDL_image SDL_ttf SDL_gfx freetype fluidsynth ]
-    ++ optionals gtkClient [ gtk3 ]
+    ++ optionals gtkClient [ gtk2 ]
```

Comment:
That won't work.

Reply:
I'm confused where that line even came from, it obviously doesn't make any sense and I can't remember adding it... My brain sometimes produces weird bugs.

Thanks for pointing out.